agent/command/s3_copy.go

```diff
@@ -188,43 +238,76 @@ func (c *s3copy) s3Copy(ctx context.Context,
 			s3CopyFile.Source.Path, s3CopyFile.Destination.Bucket,
 			s3CopyFile.Destination.Path)
 
-		s3CopyReq := apimodels.S3CopyRequest{
-			AwsKey:              c.AwsKey,
-			AwsSecret:           c.AwsSecret,
-			S3SourceRegion:      s3CopyFile.Source.Region,
-			S3SourceBucket:      s3CopyFile.Source.Bucket,
-			S3SourcePath:        s3CopyFile.Source.Path,
-			S3DestinationRegion: s3CopyFile.Destination.Region,
-			S3DestinationBucket: s3CopyFile.Destination.Bucket,
-			S3DestinationPath:   s3CopyFile.Destination.Path,
-			S3DisplayName:       s3CopyFile.DisplayName,
-			S3Permissions:       s3CopyFile.Permissions,
-		}
-
-		responseString, err := comm.S3Copy(ctx, td, &s3CopyReq)
+		s3CopyReq.AwsKey = c.AwsKey
+		s3CopyReq.AwsSecret = c.AwsSecret
 
-		if responseString != "" {
-			logger.Task().Infof("s3Copy response: %s", responseString)
+		// Now copy the file into the permanent location
```

Comment:
Same question about comments - is this one necessary?

Reply:
nah, I'll remove it